tests/system/action/test_relation_cud_required_variations.py

```diff
@@ -0,0 +1,243 @@
+        self.create_model("fake_model_a/1", {"fake_b_setnul_to_not_req_id": 1})
+        self.create_model("fake_model_b/1", {"fake_a_setnul_to_not_req_ids": [1]})
+        response = self.client.post(
+            "/",
+            json=[{"action": "fake_model_a.delete", "data": [{"id": 1}]}],
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_deleted("fake_model_a/1")
+        self.assert_model_exists("fake_model_b/1", {"fake_a_setnul_to_not_req_ids": []})
+
+
+class TestUpdateVariations(BaseActionTestCase):
+    def test_upd_a_all_to_req(self) -> None:
+        """
+        This should raise an error,
+        because the value is required in b1/fake_a_setnul_to_req_ids
+        """
+        self.create_model("fake_model_a/1", {"fake_b_setnul_to_req_id": 1})
+        self.create_model("fake_model_b/1", {"fake_a_setnul_to_req_ids": [1]})
+        response = self.client.post(
+            "/",
+            json=[{"action": "fake_model_a.update", "data": [{"id": 1, "fake_b_setnul_to_req_id": None}]}],
+        )
+        self.assert_status_code(response, 400)
+        self.assertIn("required", str(response.data))
+        self.assert_model_exists("fake_model_a/1", {"fake_b_setnul_to_req_id": 1})
+        self.assert_model_exists("fake_model_b/1", {"fake_a_setnul_to_req_ids": [1]})
+
+    def test_upd_a_all_to_not_req(self) -> None:
+        self.create_model("fake_model_a/1", {"fake_b_setnul_to_not_req_id": 1})
+        self.create_model("fake_model_b/1", {"fake_a_setnul_to_not_req_ids": [1]})
+        response = self.client.post(
+            "/",
+            json=[{"action": "fake_model_a.update", "data": [{"id": 1, "fake_b_setnul_to_not_req_id": None}]}],
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists("fake_model_a/1", {"fake_b_setnul_to_not_req_id": None})
+        self.assert_model_exists("fake_model_b/1", {"fake_a_setnul_to_not_req_ids": []})
+
+    def test_upd_a_all_to_req_multi(self) -> None:
+        self.create_model("fake_model_a/1", {"fake_b_setnul_to_req_id": 1})
+        self.create_model("fake_model_a/2", {"fake_b_setnul_to_req_id": 1})
+        self.create_model("fake_model_b/1", {"fake_a_setnul_to_req_ids": [1, 2]})
+        response = self.client.post(
+            "/",
+            json=[{"action": "fake_model_a.update", "data": [{"id": 1, "fake_b_setnul_to_req_id": None}]}],
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists("fake_model_a/1", {"fake_b_setnul_to_req_id": None})
+        self.assert_model_exists("fake_model_a/2", {"fake_b_setnul_to_req_id": 1})
+        self.assert_model_exists("fake_model_b/1", {"fake_a_setnul_to_req_ids": [2]})
+
+
+class TestCreateVariations(BaseActionTestCase):
+    def test_create_a_impossible(self) -> None:
+        """
+        Should be impossible, because fake_b_casc_to_all_id can't be set, because there is no fake_model_b instance stored.
```

Comment:
Yes, this is true, but in practice this situation will never occur, since we always have the base model `organisation/1`. All relations are rooted from there and this model is created outside of the backend-action-context. So in practice, all required relations will always be able to be set.